expr/root.go

```diff
@@ -277,3 +277,19 @@ func (m MetaExpr) Merge(src MetaExpr) {
 		}
 	}
 }
+
+// Last returns the last value for a specific key, if the key exists and
+// has values; otherwise return nil, with the "ok" flag set to false.
```

Comment:
nitpick: "otherwise returns the empty string, ..."

Reply:
Well spotted; changed.